recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "snowflake" %}
-{% set version = "0.12.1" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 383aee9dc305f6fe8fa04734fa778ecfa220da116de6a713edc30cacdc19a171
+  sha256: c47157446949ef34afc884fa6d42488e450cf3c0336ef12bed7e47f61d38ade5
   patches:
     # update 0.12.1: patch still needed
```

Comment:
```suggestion
    # update 0.13.0: patch still needed
```

Reply:
I had checked but forgot to update the comment, thank you !